elasticdl/python/master/k8s_instance_manager.py

```diff
@@ -51,6 +58,17 @@ def _parse_worker_pod_priority(num_workers, worker_pod_priority):
     return res
 
 
+def _is_pod_oom_killed(evt_obj):
```

Comment:
This is not `_is_pod_oom_killed`, but `killed_but_not_oomed`.

Reply:
Good catch, the method name is updated.